pubsub/unit_tests/test__gax.py

```diff
@@ -27,6 +27,8 @@
 
 from google.cloud._testing import _GAXBaseAPI
 
+from google.cloud.pubsub import __version__
```

Comment:
Please import this at runtime rather than at unit-test import time. (@tseaver can you link to your "philosophy of testing" blog post?)